src/components/views/UpdateAllInstalledModsModal.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <ModalCard :is-active="isOpen" :can-close="true" v-if="thunderstoreMod === null" @close-modal="closeModal()">
+        <template v-slot:header>
+            <h2 class='modal-title'>Update all installed mods</h2>
+        </template>
+        <template v-slot:body>
+            <p>All installed mods will be updated to their latest versions.</p>
+            <p>Any missing dependencies will be installed.</p>
+            <p>The following mods will be downloaded and installed:</p>
+            <br/>
+            <ul class="list">
+                <li class="list-item" v-for='(mod, index) in $store.getters["profile/modsWithUpdates"]'
+                    :key='`to-update-${index}-${mod.getFullName()}`'>
+                    {{mod.getName()}} will be updated to: {{mod.getLatestVersion()}}
+                </li>
+            </ul>
+        </template>
+        <template v-slot:footer>
+            <button class="button is-info" @click="downloadLatest()">Update all</button>
+        </template>
+    </ModalCard>
+</template>
+
+<script lang="ts">
+import { mixins } from "vue-class-component";
+import { Component } from "vue-property-decorator";
+
+import ModalCard from "../ModalCard.vue";
+import DownloadMixin from "../mixins/DownloadMixin.vue";
+import DownloadModVersionSelectModal from "../views/DownloadModVersionSelectModal.vue";
+import ThunderstoreCombo from "../../model/ThunderstoreCombo";
+import StatusEnum from "../../model/enums/StatusEnum";
+import R2Error from "../../model/errors/R2Error";
+import ThunderstoreDownloaderProvider from "../../providers/ror2/downloading/ThunderstoreDownloaderProvider";
+
+@Component({
+    components: {DownloadModVersionSelectModal, ModalCard}
+})
+export default class UpdateAllInstalledModsModal extends mixins(DownloadMixin)  {
+
+
+    async downloadLatest() {
```

Comment:
Nitpick: the naming makes little sense in the standalone component. It could be `updateAll` or `updateMods` oslt.